tests/pax/nightly/common.libsonnet

```diff
@@ -0,0 +1,61 @@
+local common = import '../common.libsonnet';
+
+{
+  NightlyPaxTest:: common.PaxTest {
+    local config = self,
+    expPath:: '',
+    extraFlags:: '',
+    buildDate:: '$(date +%Y%m%d)',
+
+    // PAX tests are structured as bash scripts that run directly on the Cloud
+    // TPU VM instead of using docker images
+    testScript:: |||
+      set -x
+      set -u
+      set -e
+
+      # .bash_logout sometimes causes a spurious bad exit code, remove it.
+      rm .bash_logout
+
+
+      # check for nightly build
+      gsutil cp gs://pax-on-cloud-tpu-project/wheels/%(buildDate)s/paxml*.whl .
+      gsutil cp gs://pax-on-cloud-tpu-project/wheels/%(buildDate)s/praxis*.whl .
+
+      if [ -f praxis*.whl -a -f paxml*.whl ]; then
+          echo "Nightly builds succeeded."
+      else 
+          echo "Nighlty builds failed or are pending."
+          exit 1
+      fi 
+
+      # install pax and dependencies
+      pip install -U pip
+      pip install praxis*.whl
+      pip install paxml*.whl
+      sudo pip uninstall --yes jax jaxlib libtpu-nightly libtpu libtpu_tpuv4 tensorflow
```

Comment:
Can you install praxis and paxml with `--no-deps` instead of uninstalling these dependencies? The `tpu-vm-base` image does not install any of these.

Reply:
sure

Reply:
the only issue is that there are a few other dependencies we would have to manually install

for pax: [
        'praxis', 'pyglove', 'absl-py', 'jax', 'tensorflow==2.7.3',
        'tensorflow-text==2.7.3', 'numpy', 'flax',
        'clu', 'lingvo', 't5', 'seqio-nightly', 'tensorstore',
        'etils', 'orbax'
    ]

for praxis: [
        'protobuf', 'absl-py', 'fiddle-config', 'numpy', 'tensorflow', 'lingvo',
        'flax', 'jax', 'optax', 'optax-shampoo', 'jax-bitempered-loss',
        'einops', 't5x', 'clu'
   ]

Reply:
@will-cromar I addressed all the feedback except this in the last commit. Should I add a step to `pip install` these dependencies `pyglove, absl-py, tensorflow, tensorflow-text, numpy, flax, clu, lingvo, t4, sequio-nightly, tensorstore, etlist, orbax, fiddle-config, optax, optax-shampoo, jax-bitempered-loss, eniops, t5x` and add the `no-deps` flags, or keep it as is?

Reply:
Discussed offline. Removing some redundant uninstalls and leaving this as a short-term fix.